feature-libs/product-configurator/rulebased/components/price/configurator-price.component.ts

```diff
@@ -41,7 +48,14 @@ export class ConfiguratorPriceComponent {
    * @return {string} - total price formula
    */
   get priceTotal(): string {
-    return '+ ' + this.formula.priceTotal?.formattedValue;
+    if (
+      this.formula.priceTotal !== undefined &&
+      this.formula.priceTotal.value >= 0
```

Comment:
The same here:
You could simply write:
`this.formula.priceTotal?.value >= 0`
This statement is not necessary `this.formula.priceTotal !== undefined` because with` this.formula.priceTotal?` will be checked  whether ` this.formula.priceTotal `is defined

Reply:
done